src/xrEngine/EngineAPI.h

```diff
@@ -60,8 +62,8 @@ class ENGINE_API CEngineAPI
 public:
     BENCH_SEC_SCRAMBLEMEMBER1
 
-    Factory_Create*  pCreate;
-    Factory_Destroy* pDestroy;
+    std::function<Factory_Create>  factoryCreate;
+    std::function<Factory_Destroy> factoryDestroy;
```

Comment:
What's the point on using std::function? My argument against using it – is that it is slower than plain pointer.